foundry/game/File.py

```diff
@@ -39,16 +61,38 @@ def __init__(self, path: Optional[str] = None):
 
         self.position = 0
 
+    @staticmethod
```

Comment:
You can't do this, since it is called from `GraphicsSet.py`, which doesn't have a `rom`.

This kind of goes to my question in the PR about why the design uses the lowercase `rom` and the static `ROM` class.

Reply:
`bank_offset` could be super misleading, considering you're passing a vanilla ROM offset and expecting to get a modified ROM offset. I don't have a great name, but I'm against `bank_offset` for this reason.

Reply:
Ah, I see my mistake.  I looked over the method and see that it actually provides the offset on an offset.  

I think we should consider a few other approaches though.  Instead of making this a static method to fix the offset, we could create an adapter class that automatically applies the offsets to the ROM class.  This would allow for the functionality inside the ROM class to remain the same while we are able to provide new features with the adapter.  It also removes the need to make a staticmethod, which I am generally against.  Furthermore, with an adapter we won't need to update existing code, only needing to change the import to use the adapter.  It also allows for future code to not need to worry about needing to know what needs to be normalized as the adapter will automatically handle the normalization.  For whatever reason, if you did not want to use the adapter, you can simply remove and do the normalization yourself.  And lastly, the adapter will be easy to test.